public/js/actions/breakpoints.js

```diff
@@ -37,10 +53,11 @@ function addBreakpoint(location, { condition, getTextForLine } = {}) {
       breakpoint: bp.toJS(),
       condition: condition,
       [PROMISE]: Task.spawn(function* () {
-        const { id, actualLocation } = yield client.setBreakpoint(
+        const res: any = yield client.setBreakpoint(
```

Comment:
This also seems unhelpful. It might be nice to type the client, but on the other hand I also don't expect it to change much and I appreciate being able to destructure on assignments.